### ISSUE TITLE
thunderbird more accurate matching

### ISSUE THREAD

**Issue by Naviigator** [kee-org/KeeFox#696]
I require exact URL matching for my thunderbird email addresses, since there are multiple accounts with multiple passwords at the domain.
Since they use different mailservers on the same domain, this can be solved by checking the entire URL instead of just the domain.

It seemed that "more accurate URL matching" was the sollution, but that one also seems to ignore the URL and go straight for the domain.

in short:
a.domain.something => hits one entry
b.domain.something => hits another entry

**Comment by dlech** [comment]
Do you include the url prefix? For example, `imap://a.domain.something`.

**Comment by Naviigator** [comment]
Yes, I did, but just 1 entry was selected each time. (and used as password for all addresses of domain.something)

**Comment by dlech** [comment]
Due to the recent split between Thunderbird and Firefox extensions, we have started a fork of KeeFox specifically for Thunderbird. We are calling this new extension KeeBird and the code and issue tracking are now in a new GitHub repository. If anyone is still interested in getting this issue resolved, please let us know by opening a new issue at https://github.com/kee-org/keebird/issues and copy/paste any relevant information (or at least link to this issue).
